Chess/Assets/Scripts/ArrayChess/PieceMover.cs

```diff
@@ -16,24 +14,21 @@ public class PieceMover : MonoBehaviour
 
     Ray ray;
     RaycastHit hit;
-    private string rayHitStringParent;
+    private string mouseClickPosition;
     private string rayHitStringChild;
     private GameObject rayHitGameObject;
 
 
     private Renderer rend;
-    private Renderer pieceSquareRend;
```

Comment:
I think a lot of these renaming is just for readability, I can understand why its called we're using Rend in the naming, but it doesn't seem that beneficial codewise imo, but I could be convinced otherwise...